Platform/CoffeelakeBoardPkg/BoardConfig.py

```diff
@@ -49,7 +49,10 @@ def __init__(self, *args, **kwargs):
 		self.ENABLE_SPLASH            = 1
 		self.ENABLE_FRAMEBUFFER_INIT  = 1
 		self.HAVE_PSD_TABLE       = 1
-		self.ENABLE_GRUB_CONFIG       = 1
+		self.ENABLE_GRUB_CONFIG   = 1
+		self.ENABLE_CSME_UPDATE   = 0
+
```

Comment:
add a comment here to mention a library is required and it is in the TXE kit.

Reply:
Done